modules/debts_api/app/sidekiq/debts_api/v0/form5655/vha_submission_job.rb

```diff
@@ -13,8 +13,16 @@ class V0::Form5655::VHASubmissionJob
     class MissingUserAttributesError < StandardError; end
 
     sidekiq_retries_exhausted do |job, _ex|
+      StatsD.increment("#{STATS_KEY}.retries_exhausted")
+      submission_id = job['args'][0]
       user_uuid = job['args'][1]
-      UserProfileAttributes.find(user_uuid)&.destroy
+      Rails.logger.error <<~LOG
+        V0::Form5655::VHASubmissionJob retries exhausted:
+        Exception: #{ex.class} - #{ex.message}
+        Backtrace: #{ex.backtrace.join("\n")}
+        submission_id: #{submission_id} | user_id: #{user_uuid}
+      LOG
+      UserProfileAttributes.find(user_uuid)&.destroy # TODO: figure out why are we destroying here.
```

Comment:
I would recommend understand the "# TODO: figure out why are we destroying here." before committing this

Reply:
Oops left the comment out. When we took a look at this, the destroy method is an optimization for Redis cache which marks the entry as deleted and prevents persistence.